wordpress-mobile/WordPress-Android
# Feature/populate domain info phone prefix when selecting country

This PR adds automatic phone country code selection when a country is selected from the picker.

To test:
- Switch to a site with unredeemed domain credit.
- Click on Domain Registration CTA, and pick a domain name.
- On domain registration details select a country.
- Notice that phone number's country code is filled with correct country code.
- Rejoice!

Update release notes:
